sources/zafira-services/src/main/java/com/qaprosoft/zafira/service/GroupService.java

```diff
@@ -120,6 +127,13 @@ public Group updateGroup(Group group) {
     @CacheEvict(value = "groups", key = "new com.qaprosoft.zafira.dbaccess.utils.TenancyContext().getTenantName() + ':' + #id")
     @Transactional(rollbackFor = Exception.class)
     public void deleteGroup(long id) {
```

Comment:
Let's perform group null check in `getGroupById` service method, and omit null checks wherever we use it.